_static/css/aiven.css

```diff
@@ -1,32 +1,8 @@
 /* Assigning color variables */
```

Comment:
--linkActive is no longer red, in the dark mode the hover on the menu item loses red side line. Based on other changes it should be probably changed with --color-link.

.sidebar-tree .reference:hover {
  background: none;
  border-left: 1px solid var(--linkActive);
  font-family: Inter;
}
.sidebar-tree .current-page > .reference {
  background: none !important;
  border-left: 1px solid var(--linkActive);
  color: var(--linkActive) !important;
  font-weight: 400 !important;
}

Reply:
Additionally, there is 1px movement on hover, which is not needed. This might be related to the fact that the border-left is gone

Reply:
I didn't see the hover! I think I have this looking better but let me know what you think (when I push the commit...)